spec/services/proforma_service/convert_task_to_exercise_spec.rb

```diff
@@ -187,6 +187,27 @@
         expect { convert_to_exercise_service.save! }.to change(Exercise, :count).by(1)
       end
 
+      context 'with two files with similar contents' do
```

Comment:
I apologize for my lack of understanding. Should we not ensure that the exported file has no duplicates?  Won't we have the same issue in CodeHarbor after exporting the it from CodeHarbor and importing it again to CodeHarbor?

Reply:
Are the duplicates some requirement in the ProformaXML spec?

Reply:
It would be preferable to also fix the export of CodeHarbor, since I only addressed the import of tasks with multi-referenced files and not the export.

It would be even better to enable CodeHarbor to support the multi-referenced files directly (which would require a serious time investment).